app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,66 @@
 package com.codesoom.assignment;
 
+import com.codesoom.assignment.handlers.RootHttpHandler;
+import com.codesoom.assignment.handlers.TaskHttpHandler;
+import com.google.common.primitives.Bytes;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
 public class App {
+
+    public static final int PORT = 8000;
+    public static final int BACKLOG = 0;
+
+    /* Http Status Code 를 정의한 Enum */
```

Comment:
JavaDoc 은 * 두개로 시작합니다.
한편 `Enum`을 설명할 때 `Enum`이라 쓰는 것은 불필요합니다.
```suggestion
    /** Http Status Code 를 정의합니다. */
```